virt/install/preparing-cluster-for-virt.adoc

```diff
@@ -9,6 +9,13 @@ Before you install {VirtProductName}, ensure that your {product-title} cluster m
 * Your cluster must be installed on
 xref:../../installing/installing_bare_metal/installing-bare-metal.adoc#installing-bare-metal[bare metal] infrastructure with Red Hat Enterprise Linux CoreOS workers.
 
+* Additionally, your cluster must use xref:../../installing/installing_bare_metal_ipi/ipi-install-overview.adoc#ipi-install-overview[Installer-provisioned installation] and xref:../../machine_management/deploying-machine-health-checks.adoc#machine-health-checks-about_deploying-machine-health-checks[deploy machine health checks] to maintain High Availability (HA).
```

Comment:
* Lowercase "Installer" since it's in running text, not the name of a section
* Also, it should be called "installer-provisioned infrastructure" (not installation)
* High Availability (HA) -> high availability (HA)